lute/cli/commands/lint/init.luau

```diff
@@ -89,6 +91,52 @@ local function loadDefaultRules(): { types.LintRule }
 	return rules
 end
 
+local function violationIsSuppressed(
+	violation: types.LintViolation,
+	ast: syntax.AstStatBlock,
+	suppressionCache: { [syntax.AstNode]: { string } }
+): boolean
+	local violationSuppressed = false
+
+	local function nodeIsSuppressed(node: syntax.AstNode): boolean
+		if syntax.span.subsumes(node.location, violation.location) then
+			local rules = suppressionCache[node]
+			if rules == nil then
+				-- Find all rules that this node suppresses
+				local leadingTrivia = triviaUtils.leftmosttrivia(node)
+				rules = {}
+				for _, trivia in leadingTrivia do
+					if trivia.tag == "blockcomment" or trivia.tag == "comment" then
+						for rule in trivia.text:gmatch("lute%-lint%-ignore%((.+)%)") do
+							print(`Found suppression for rule '{rule}'`)
```

Comment:
I would not include the print statement here, though maybe it's time to add a logging library to the batteries

Reply:
sg, will do